src/popup/components/NameItem.vue

```diff
@@ -26,12 +26,12 @@
         >
           <button
             v-show="canBeDefault"
-            :class="{ set: account.name === name }"
-            :disabled="account.name === name"
+            :class="{ set: activeAccount.name === name }"
+            :disabled="activeAccount.name === name"
             @click="setDefault"
           >
             {{
-              account.name === name
+              activeAccount.name === name
```

Comment:
That comparison is made three times here - what are your thoughts of a computed property that will check that?

Reply:
OK, I created `isDefault` computed property for this purpose.